src/kohlrahbi/conditions/command.py

```diff
@@ -14,16 +14,46 @@
 To use this command-line interface, run the script and provide the necessary options as command-line arguments.
 
 Example:
-$ python command.py --edi-energy-mirror-path /path/to/edi_energy_mirror/repo
- --output-path /path/to/output --format-version FV2310 --assume-yes
+$ python command.py --input-path /path/to/input --output-path /path/to/output --format-version FV2310 --assume-yes
```

Comment:
`--input-path` should be `edi-energy-mirror-path` doesn't it?

Reply:
59127c985acd8f23c52090a0a87197a439ce9636